python/users.py

```diff
@@ -17,6 +17,19 @@
 from sdk.python.domains import EnrolledDomain
 
 
+class SystemRole(sdk.python.entities.Entity):
```

Comment:
should maybe have it's own file?

Reply:
I think put them in file is better, they are ... too close

Reply:
it does make it more difficult to find the definition.